make/libraries

```diff
@@ -19,17 +19,9 @@ SUNDIALS_NVECSERIAL := $(patsubst %.c,%.o,\
   $(addprefix $(CVODES)/src/, nvec_ser/nvector_serial.c sundials/sundials_math.c))
 
 
-$(sort $(SUNDIALS_CVODES) $(SUNDIALS_NVECSERIAL)) : %.o : %.c
-	@mkdir -p $(dir $@)
-	$(COMPILE.c) -x c -O$(O) -isystem $(CVODES)/include $< -o $@
-
-$(CVODES)/lib/libsundials_cvodes.a: $(SUNDIALS_CVODES)
-	@mkdir -p $(dir $@)
-	$(AR) -rs $@ $^
-
-$(CVODES)/lib/libsundials_nvecserial.a: $(SUNDIALS_NVECSERIAL)
-	@mkdir -p $(dir $@)
-	$(AR) -rs $@ $^
+$(SUNDIALS_CVODES) $(SUNDIALS_NVECSERIAL) : CFLAGS += -isystem $(CVODES)/include -x c
```

Comment:
To clarify the language a little, I think you're using implicit rules here, which in turn use some predefined macros (like `COMPILE.c`). Maybe there are no "implicit macros" after all, haha.
I've found this document to be the most helpful in this area: http://docs.oracle.com/cd/E19504-01/802-5880/6i9k05dhg/index.html